libs/vm-modules/src/ml/model/model_estimator.cpp

```diff
@@ -65,8 +65,8 @@ ModelEstimator &ModelEstimator::operator=(ModelEstimator &&other) noexcept
 ChargeAmount ModelEstimator::LayerAddDense(Ptr<String> const &layer, SizeType const &inputs,
                                            SizeType const &hidden_nodes)
 {
-  SizeType size{0};
-  SizeType padded_size{0};
+  DataType size{0};
```

Comment:
Don't think there's any point making any changes to vmmodules/ml estimators. These will all be ditched in favour of a better mechanism that @vhavrylov-ncube is working on

Reply:
done